tests/e2e/models/onboarding/BusinessSection.ts

```diff
@@ -0,0 +1,68 @@
+import { Page } from 'playwright';
+import { DropdownField } from '../DropdownField';
+
+export class BusinessSection {
+	page: Page;
+	howManyProductsDropdown: DropdownField;
+	sellingElsewhereDropdown: DropdownField;
+
+	constructor( page: Page ) {
+		this.page = page;
+		this.howManyProductsDropdown = new DropdownField(
+			page,
+			':nth-match(.woocommerce-select-control, 1)'
+		);
+		this.sellingElsewhereDropdown = new DropdownField(
+			page,
+			':nth-match(.woocommerce-select-control, 2)'
+		);
+	}
+
+	async isDisplayed() {
+		await this.page.waitForSelector(
+			':text("Tell us about your business")'
+		);
+	}
+
+	async selectProductNumber( productLabel: string ) {
+		await this.howManyProductsDropdown.select( productLabel );
+	}
+
+	async selectCurrentlySelling( currentlySelling: string ) {
+		await this.sellingElsewhereDropdown.select( currentlySelling );
+	}
+
+	async selectInstallFreeBusinessFeatures( select: boolean ) {
+		const installFeaturesCheckbox = await this.page.$(
+			'#woocommerce-business-extensions__checkbox'
+		);
+		if ( select ) {
+			await installFeaturesCheckbox?.check();
+		} else {
+			await installFeaturesCheckbox?.uncheck();
+		}
+	}
+
+	async expandRecommendedBusinessFeatures() {
+		const expandButtonSelector =
+			'.woocommerce-admin__business-details__selective-extensions-bundle__expand';
+		await this.page.waitForSelector( expandButtonSelector );
```

Comment:
I won't mark every one of these examples, but here is another one, anywhere we interact with something we were waiting on we should remove `waitForSelector`